packages/strapi-plugin-users-permissions/controllers/User.js

```diff
@@ -117,14 +117,6 @@ module.exports = {
         key: 'advanced'
       }).get();
 
-      if (advancedConfigs.unique_email && ctx.request.body.email) {
```

Comment:
Hello @alphapseudo , can you tell me why you delete this condition please.

Reply:
Hi @lauriejim, sorry for not getting back to you. Just for future reference, the block at line 120 is performing the same check and assertion as the block and condition at line 138.